### PR TITLE
Update Log driver example to xstream

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: _posts/2015-01-23-drivers.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Drivers should focus solely on being an interface for effects, and usually are l
 
 {% highlight js %}
 run(main, {
-  log: msg$ => { msg$.subscribe(msg => console.log(msg)) }
+  log: msg$ => { msg$.addListener({next: msg => console.log(msg)}) }
```
Comment (mightyiam) [inline _posts/2015-01-23-drivers.md:92]: In xstream, if I remember correctly, you must provide all three, `next`, `error` and `complete` properties.

**Review by mightyiam — CHANGES_REQUESTED** [review]
Just my two cents.

File: _posts/2015-01-23-drivers.md (hunk 1 above)
Comment (SteveALee) [reply _posts/2015-01-23-drivers.md:92]: @staltz recently said on Gitter that only next is required. I've not double checked the code though. Thanks for comment.

**Comment by staltz** [comment]
Correct. Thanks @SteveALee !

**Pull request merged**